core/ui/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can remove this file unless there's a specific reason it's needed for exposing resources.